src/pages/journal/components/empty-slot/empty-slot.scss

```diff
@@ -0,0 +1,47 @@
+empty-slot {
+  ion-card {
+    &.grey-bg {
+      background: transparent;
+    }
+    border: 4px dashed #4A4A4A;
+    ion-grid {
+      height: 100%;
+    }
+  }
+  ion-row {
+    &.slot-row {
+      min-height: 108px;
+      position: relative;
+      &.test-slot-portrait-mode {
+        min-height:108px;
+      }
+    }
+  }
+  .time-column {
+    max-width: 152px;
+    padding-left: 55px;
+  }
+  .available-column {
+    display: flex;
+    justify-content: center;
+    .available {
+      font-weight: bold;
+    }
+  }
+  .spacing-column {
+    max-width: 152px;
+    display: flex;
+    justify-content: center;
+    .activity-code {
+      font-weight: bold;
+    }
+  }
+  .slot-changed-indicator {
```

Comment:
Are these bits about slot change still relevant to an empty slot?

Reply:
removed